Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -818,26 +829,20 @@ extension SourceKitLSPServer {
 
   /// Creates a workspace at the given `uri`.
   ///
-  /// If the build system that was determined for the workspace does not satisfy `condition`, `nil` is returned.
+  /// A workspace does not necessarily have any buildsystem attached to it, in which case `buildSystemSpec` may be
+  /// `nil` - consider eg. a top level workspace folder with multiple SwiftPM projects inside it.
   private func createWorkspace(
     workspaceFolder: DocumentURI,
+    options: SourceKitLSPOptions,
     buildSystemSpec: BuildSystemSpec?
   ) async throws -> Workspace {
     guard let capabilityRegistry = capabilityRegistry else {
       struct NoCapabilityRegistryError: Error {}
       logger.log("Cannot open workspace before server is initialized")
       throw NoCapabilityRegistryError()
     }
-    let testHooks = self.testHooks
-    let options = SourceKitLSPOptions.merging(
-      base: self.options,
-      override: SourceKitLSPOptions(
-        path: workspaceFolder.fileURL?
-          .appendingPathComponent(".sourcekit-lsp")
-          .appendingPathComponent("config.json")
-      )
-    )
-    logger.log("Creating workspace at \(workspaceFolder.forLogging) with options: \(options.forLogging)")
+
+    logger.log("Creating workspace at \(workspaceFolder.forLogging)")
```

Comment:
Did you move the logging of `options` somewhere else? If not, I would prefer to continue logging them. I had a case once where logging them was crucial to figure out a bug.

Reply:
It's logged in its entirety in the line below this